src/entities/tag.cpp

```diff
@@ -469,7 +482,8 @@ bool Tag::store() {
     if (this->id > 0) {
         query.prepare(
                 "UPDATE tag SET name = :name, priority = :priority, "
-                        "parent_id = :parentId, " + colorField + " = :color "
+                        "parent_id = :parentId, " + colorField + " = :color, "
+                        "created = datetime('now') "
```

Comment:
Yeah, i somewhat abuse that a bit. But I didn't see it in use so far and it seemed perfect for what I wanted to do. One can also argue, that if you rename a tag for instance, you basically create a new one with the same id while deleting the old one, now there's only confusion, when updating a child tag :D. Introducing another column didn't seem that great for me, because i think it will mostly hold duplicate data (except if you make heavy use of managing your tags in a tree, or keep renaming them all the time). I would rather vote to rename created to updated.